pep-0669.rst

```diff
@@ -1,12 +1,14 @@
 PEP: 669
 Title: Low Impact Monitoring for CPython
 Author: Mark Shannon <mark@hotpy.org>
-Status: Draft
+Discussions-To: https://discuss.python.org/t/pep-669-low-impact-monitoring-for-cpython/13018/
+Status: Accepted
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 18-Aug-2021
```

Comment:
Also, one thing I forgot to mention before, sorry: this should presumably have a `Python-Version`; assuming this is still targeted for 3.12, it should thus be

```suggestion
Created: 18-Aug-2021
Python-Version: 3.12
```